src/main/java/entity/Calendar.java

```diff
@@ -14,7 +14,7 @@ public Calendar(String name, List<Event> events) {
         TodoList todoList = new TodoList();
 
         for(Event event : events) {
-            Task task = new Task(event.getId(), event.getEventName());
+            Task task = new Task(event.getId(), "");
```

Comment:
why are we making the task name an empty string? we can give it a default name as being the same as the associated event.

Reply:
ah that's because the event entity itself doesn't hold its name anymore, and the calendar doesn't even do anything now so I just arbitrarily changed the value.